containers/init/ready/ready_test.go

```diff
@@ -61,15 +62,24 @@ var _ = Describe("WaitForReadyPods", func() {
 		defer cancel()
 
 		podListerMock := &PodListerMock{
-			PodList: &corev1.PodList{},
+			PodList: &corev1.PodList{
+				Items: []corev1.Pod{driverPod},
+			},
 		}
 
 		loadTestGetterMock := &LoadTestGetterMock{
 			Loadtest: newLoadTestWithMultipleClientsAndServers(0, 0),
 		}
-		podAddresses, err := WaitForReadyPods(ctx, loadTestGetterMock, podListerMock, "test name")
+		podAddresses, nodesInfo, err := WaitForReadyPods(ctx, loadTestGetterMock, podListerMock, "test name")
 		Expect(err).ToNot(HaveOccurred())
 		Expect(podAddresses).To(BeEmpty())
+		Expect(*nodesInfo).To(Equal(NodesInfo{
+			Driver: NodeInfo{
+				Name:     driverPod.Name,
+				PodIP:    net.JoinHostPort(driverPod.Status.PodIP, "10000"),
```

Comment:
In general, I think PodIP should be just the IP address, not a port.

Reply:
Make sense, done.